apps/api/src/app/subscribers-v2/subscribers.controller.ts

```diff
@@ -136,11 +136,15 @@ export class SubscribersController {
       **subscriberId** is a required field, rest other fields are optional, if the subscriber already exists, it will be updated`,
   })
   @ApiResponse(SubscriberResponseDto, 201)
+  @ApiResponse(SubscriberResponseDto, 409, false, false, {
+    description: 'Subscriber already exists (when query param failIfExists=true)',
+  })
   @SdkMethodName('create')
   @RequirePermissions(PermissionsEnum.SUBSCRIBER_WRITE)
   async createSubscriber(
     @UserSession() user: UserSessionData,
-    @Body() body: CreateSubscriberRequestDto
+    @Body() body: CreateSubscriberRequestDto,
+    @Query('failIfExists') failIfExists?: boolean
```

Comment:
the new query param that controls whether it should throw 409 when the subscriber exists on creation; 
similar changes are made to the topics create endpoint;